contracts/HubPool.sol

```diff
@@ -42,48 +49,70 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     // `slowRelayRoot` to a SpokePool. The latter two roots, once published to the SpokePool, contain
     // leaves that can be executed on the SpokePool to pay relayers or recipients.
     struct RootBundle {
+        // When root bundle challenge period passes and this root bundle becomes executable.
         uint64 requestExpirationTimestamp;
+        // Number of pool rebalance leaves to execute in the `poolRebalanceRoot`. After this number
+        // of leaves are executed, a new root bundle can be proposed
         uint64 unclaimedPoolRebalanceLeafCount;
+        // Contains leaves instructing this contract to send funds to SpokePools.
         bytes32 poolRebalanceRoot;
+        // Relayer refund merkle root to be published to an L2.
         bytes32 relayerRefundRoot;
+        // Slow relay merkle root to be published to an L2.
         bytes32 slowRelayRoot;
-        uint256 claimedBitMap; // This is a 1D bitmap, with max size of 256 elements, limiting us to 256 chainsIds.
+        // This is a 1D bitmap, with max size of 256 elements, limiting us to 256 chainsIds.
+        uint256 claimedBitMap;
+        // Proposer of this root bundle.
         address proposer;
+        // Whether bond has been repaid to successful root bundle proposer.
         bool proposerBondRepaid;
     }
 
+    // Only one root bundle can be stored at a time. Once all pool rebalance leaves are executed, a new proposal
+    // can be submitted.
     RootBundle public rootBundleProposal;
 
-    // Whitelist of origin token to destination token routings to be used by off-chain agents. The notion of a route
-    // does not need to include L1; it can store L2->L2 routes i.e USDC on Arbitrum -> USDC on Optimism as a "route".
+    // Whitelist of origin token + ID to destination token routings to be used by off-chain agents. The notion of a
+    // route does not need to include L1; it can store L2->L2 routes i.e USDC on Arbitrum -> USDC on Optimism as a
+    // "route".
```

Comment:
```suggestion
    // Whitelist of origin token + ID to destination token routings to be used by off-chain agents. The notion of a
    // route does not need to include L1; it can be L2->L2 route. i.e USDC on Arbitrum -> USDC on Optimism as a "route".
```

let's try not to wrap onto new lines for one word.